samples/cloudwatch/README.md

```diff
@@ -1,11 +1,11 @@
 # Amazon CloudWatch Integration
 
 This sample describes how to configure Amazon CloudWatch to collect metrics
-from the services and components in SAS Viya. Complete the deployment
+from the services and components in SAS Viya platform platform. Complete the deployment
```

Comment:
"platform platform" is almost certainly incorrect.  However, see comment above re: scope of instrumentation and monitoring.